### ISSUE TITLE
Question on stability of this library

### ISSUE THREAD

**Issue by s1nisteR** [lexbor/lexbor#240]
Hello. Thank you for lexbor. I wanted to ask how stable this library is when encountering malformed HTML. Would it match the stability of something like libxml2? I am planning to use it to scrape Tor(Onion websites) hence I'm concerned about how it would perform in the face of malformed html.

**Comment by lexborisov** [comment]
Hello @s1nisteR 

This library is fully compliant with the requirements of modern HTML. The library complies with the [live HTML specification](https://html.spec.whatwg.org/multipage/). In other words, it parses HTML in the same way as modern browsers, because it is written according to the same specification.

Unfortunately, the libxml2 library is not compliant with the standard and parses HTML as it sees fit (possible exception is HTML 4).

In other words, like Tor, Chrome, FireFox and all modern browsers, lexbor library parse HTML the same way.

**Comment by kostya** [comment]
@s1nisteR I don't know what you mean by stability of libxml2, for me it had many problems, from bugs to segfaults. And no problems with lexbor.

**Comment by s1nisteR** [comment]
Thank you everyone. Currently on to using lexbor for my application. It just raised my confidence in using this library for my use-case.
